src/ophyd_async/epics/_backend/_aioca.py

```diff
@@ -135,6 +135,14 @@ def write_value(self, value: str):
         return value + "\0"
 
 
+class CaArrayConverter(CaConverter):
+    def get_datakey(self, source: str, value: AugmentedValue) -> DataKey:
+        return {"source": source, "dtype": "array", "shape": [len(value)]}
+
```

Comment:
```suggestion
```

We can still use the same handling for the datakey

Reply:
I just removed this?